packages/react-select/src/components/Select/useSelectStyles.ts

```diff
@@ -12,36 +13,197 @@ export const selectClassNames: SlotClassNames<SelectSlots> = {
   icon: 'fui-Select__icon',
 };
 
-const useStyles = makeStyles({
-  wrapper: {
-    // TODO: add styles
+// TODO(sharing) use theme values once available
+const horizontalSpacing = {
+  xxs: '2px',
+  xs: '4px',
+  sNudge: '6px',
+  s: '8px',
+  mNudge: '10px',
+  m: '12px',
+};
+const contentSizes = {
+  // TODO: borrowed this from Input, should be in the theme somewhere?
+  body1: {
+    fontSize: tokens.fontSizeBase300,
+    lineHeight: tokens.lineHeightBase300,
+  },
+  caption1: {
+    fontSize: tokens.fontSizeBase200,
+    lineHeight: tokens.lineHeightBase200,
   },
-  select: {
-    // TODO: add styles
+  400: {
+    fontSize: tokens.fontSizeBase400,
+    lineHeight: tokens.lineHeightBase400,
   },
-  selectDisabled: {
-    // TODO: add styles
+};
+// TODO: borrowed this from Input (Select has the same size in design comps)
+// Should be in the theme somewhere?
+const fieldHeights = {
+  small: '24px',
+  medium: '32px',
+  large: '40px',
+};
+
+// TODO: borrowed from Input, also seems like animation should be included in the theme:
+const motionDurations = {
+  ultraFast: '0.05s',
```

Comment:
Having decimal seconds reads strange rather than specifying in milliseconds, but maybe I'm old school CSS here.

Reply:
I borrowed these wholesale from Input so we could rip them out and put them somewhere else easily :)

I actually kinda do like putting all animation durations in seconds b/c I've found it easier to compare values across different durations. That's a super light personal preference though, and I don't actually care much whether we use `s` or `ms` :D